bindings/python/src/datetime.cpp

```diff
@@ -55,6 +56,33 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point tpt)
+    {
+        object result;
+        if (tpt > lt::min_time()) {
+            lt::time_point now = lt::clock_type::now();
+            time_t const tm = lt::total_seconds(tpt - now);
```

Comment:
Oh damn I forgot I removed that `time(NULL)`...

I guess the `system_clock` version was working so yeah. Is there a quick way to get a proper `next_announce` value when testing?